packages/@aws-cdk/assertions/README.md

```diff
@@ -0,0 +1,135 @@
+# Assert v2
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![cdk-constructs: Experimental](https://img.shields.io/badge/cdk--constructs-experimental-important.svg?style=for-the-badge)
+
+> The APIs of higher level constructs in this module are experimental and under active development.
+> They are subject to non-backward compatible changes or removal in any future version. These are
+> not subject to the [Semantic Versioning](https://semver.org/) model and breaking changes will be
+> announced in the release notes. This means that while you may use them, you may need to update
+> your source code when upgrading to a newer version of this package.
+
+---
+
+<!--END STABILITY BANNER-->
+
+This module allows asserting the contents of CloudFormation templates.
```

Comment:
I think that in the future we might also want to support creating assertions against things like cloud assemblies, assets, etc, so I wouldn't want to scope this down too much at the outset:

```suggestion
Functions for writing test asserting against CDK applications, with focus on CloudFormation templates.
```